docker/build-git-openssl.sh

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env bash
+#
+# Run this script on x86 and aarch Ubuntu 16.04 hosts to generate binary git
+# packages built with openssl instead of gnutls. Then copy the main git deb
+# files into the docker/lfs directory of this repo, and rename them to end
+# with 'x86_64.deb' or 'aarch64.deb'.
```

Comment:
Doing the multistage build I think makes more sense I think. If not it would be nice to have a `rebuild.sh` or similar that does all these steps but run in a docker container, like:
```bash
case $arch in
x86_64) image=ubuntu:16.04;;
aarch64) image=the-ubuntu-qemu-image;;
*) error
esac

docker run --rm -ti -v $PWD:$PWD -w $PWD $image ./rebuild.sh
```